charts/mysqldump/values.yaml

```diff
@@ -5,12 +5,14 @@
 image:
   registry: quay.io
   repository: monotek/gcloud-mysql
-  tag: "master-13"
+  tag: "master-14"
   ## Specify a imagePullPolicy
   ## Defaults to 'Always' if image tag is 'latest', else set to 'IfNotPresent'
   ## ref: http://kubernetes.io/docs/user-guide/images/#pre-pulling-images#
   pullPolicy: IfNotPresent
 
+sshMountpath: "/home/cloudsdk"
+
```

Comment:
Please keep the default.

Reply:
So i need to set master-13 as default in other files ? otherwise the SSH key will be put in root but the cloudsk user used in master-14 will not be able to use it.

Reply:
You mean for pullPolicy or for sshMountPath ? If you mean sshMountPath set to /root, i must replace default image tag to master-13, otherwise the SSH key will be put in root but the cloudsk user used in master-14 will not be able to use it, and if you mean pullPolicy, it's IfnotPresent because i didn't use the latest (when we were in latest version, mysqldump stop working because of user change in dockerfile by cloudsdk, so i've made master-14 as default to not be trapped again)